### PR TITLE
Ddpb 2669 saaa gifts

### PR DESCRIPTION
**Pull request by slizzio**
## Purpose
Add a Save and Add another to gifts, replacing the need to visit an unnecessary page in journey

Fixes [DDPB-2669]

## Approach
Used existing macros to add the buttons and amended the redirects depending on which button is pressed.

## Checklist
- [x] I have performed a self-review of my own code
- [ ] I have updated documentation (Confluence/GitHub wiki) where relevant
- [x] I have added tests to prove my work, and they follow our [best practices](https://github.com/ministryofjustice/opg-digi-deps-client/wiki/Testing-best-practices)
- [x] I have successfully built my branch to a feature environment
- [x] New and existing unit tests pass locally with my changes (`docker-compose run --rm test sh scripts/clienttest.sh`)
- [ ] There are no new frontend linting errors (`docker-compose run --rm npm run lint`)
- [ ] There are no NPM security issues (`docker-compose run --rm npm audit`)
- [ ] There are no Composer security issues (`docker-compose run frontend php app/console security:check`)
- [ ] The product team have tested these changes

### CODE REVIEW

File: src/AppBundle/Resources/views/Report/Gift/_form.html.twig [hunk 1]
```diff
@@ -10,6 +12,12 @@
         {{ form_select(form.bankAccountId, 'form.bankAccount') }}
     {% endif %}
 
-    {{ macros.saveAndContinueButton(form.save) }}
+    {{ macros.saveAndContinueButton(form.saveAndContinue) }}
+
+    {% if form.saveAndAddAnother is defined %}
+        <span class="push-half--sides text--center">or</span>
```
Comment (gregtyler) [inline src/AppBundle/Resources/views/Report/Gift/_form.html.twig]: This is using old GOV.UK Elements classes. I believe the classes should be `govuk-!-margin-left-3 govuk-!-margin-right-3`. I don't think `text--center` would do anything useful in this case.

Comment (gregtyler) [reply src/AppBundle/Resources/views/Report/Gift/_form.html.twig]: Sorry, correction: This is using some custom DigiDeps classes which are now part of the Design System, so we should use the Design System versions instead. `govuk-!-margin-left-3 govuk-!-margin-right-3` should do it.

Comment (slizzio) [reply src/AppBundle/Resources/views/Report/Gift/_form.html.twig]: will fix. Ta

Comment (slizzio) [reply src/AppBundle/Resources/views/Report/Gift/_form.html.twig]: Fixed, applied gov uk style classes to macros and removed span altogether

**Pull request merged**